src/redux/books/booksSlice.js

```diff
@@ -1,26 +1,39 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const BASE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/wrcRoQ7axFGLY3A38Uya/books';
+
+export const getBooks = createAsyncThunk('books/getBooks', async () => {
+  try {
+    const response = await axios.get(BASE_URL);
+    return response.data;
+  } catch (err) {
+    return err.message;
+  }
+});
```

Comment:
As part of the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connect_to_api.md#:~:text=Create%20a%20special,from%20inside%20useEffect), you are suppose to 

- [ ] create a special action creator using createAsyncThunk called `fetchBooks`, 
- [ ] add `fetchBooks` to the extraReducers section of your booksSlice, and 
- [ ] dispatch the `fetchBooks` from inside useEffect. 
However, I do not see `fetchBooks` in your booksSlice. Kindly update your code to reflect this requirement since it is a special requirement that cannot be overlooked. :pray:

Reply:
Thank you for the review. Will effect changes.